File: 백준/Bronze/20492. 세금/세금.cs
using System;

class Q20492 
{
    static void Main(string[] args)
    {
        double N = double.Parse(Console.ReadLine());
        double No1 = N * 0.78;
        double No2 = N * 0.8 + N * 0.2 * 0.78;
        
        Console.WriteLine((int)No1 + " " + (int)No2);
    }
}

File: 백준/Bronze/20492. 세금/세금.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        double N = sc.nextInt();
        double No1 = N * 0.78;
        double No2 = N * 0.8 + N * 0.2 * 0.78;
        
        System.out.println((int)No1 + " " + (int)No2);
    }
}